.gitignore

```diff
@@ -17,6 +17,9 @@ poetry.lock
 # pycharm
 .idea/
 
+# pyright
+.pyright/
```

Comment:
I configured `pyright` to save its environment in the project's directory. That way it can persist between runs (since it is outside Docker) and it doesn't have to rebuild every time